interfaces/builtin/rawusb.go

```diff
@@ -0,0 +1,43 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const rawusbConnectedPlugAppArmor = `
+# Until we have proper device assignment, allow access to all usb devices
+/dev/bus/usb/[0-9]{3}/[0-9]{3} rw,
+
+# Allow detection of usb devices. Leaks plugged in USB device info
+/sys/bus/usb/devices/ r,
+/sys/devices/pci*/*/usb*/**/idVendor r,
+/sys/devices/pci*/*/usb*/**/idProduct r
+`
+
+// NewRawUsbInterface returns a new "rawusb" interface.
+func NewRawUsbInterface() interfaces.Interface {
+	return &commonInterface{
+		name: "raw-usb",
```

Comment:
We have a 'hiddraw' interface (I realize that is a name from the kernel), so 'raw-usb' sounds a bit inconsistent to me. What about 'usbraw'? @niemeyer, can you also weigh in on the naming of this transitional interface?